LICENSE.md

```diff
@@ -1,5 +1,14 @@
-THE NATIONAL TELECOMMUNICATIONS AND INFORMATION ADMINISTRATION, INSTITUTE FOR TELECOMMUNICATION SCIENCES ("NTIA/ITS") DOES NOT MAKE ANY WARRANTY OF ANY KIND, EXPRESS, IMPLIED OR STATUTORY, INCLUDING, WITHOUT LIMITATION, THE IMPLIED WARRANTY OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT AND DATA ACCURACY. THIS SOFTWARE IS PROVIDED "AS IS." NTIA/ITS does not warrant or make any representations regarding the use of the software or the results thereof, including but not limited to the correctness, accuracy, reliability or usefulness of the software or the results.
+SOFTWARE DISCLAIMER / RELEASE
 
-You can use, copy, modify, and redistribute the NTIA/ITS developed software upon your acceptance of these terms and conditions and upon your express agreement to provide appropriate acknowledgments of NTIA's ownership of and development of the software by keeping this exact text present in any copied or derivative works.
+This software was developed by employees of the National Telecommunications and Information Administration, an agency of the Federal Government and is provided to you as a public service.  Pursuant to title 15 United States Code Section 105, works of NTIA employees are not subject to copyright protection within the United States.
```

Comment:
Should NTIA be put in parenthesis after National Telecommunications and Information Administration? For example, "developed by employees of the National Telecommunications and Information Administration (NTIA)," 

Should "title" be capitalized?  For example, "Title 15 United States Code Section 105".

Reply:
Yes, to both.

Reply:
Changes applied